knowledge-base/amannn/action-semantic-pull-request/action-security.yml

```diff
@@ -0,0 +1,9 @@
+name: 'semantic-pull-request'
+github-token:
+  environment-variable-name: GITHUB_TOKEN
+  is-default: true
```

Comment:
`is-default` should not be set for env variable. @Devils-Knight please see if you can add a test for this.

Reply:
@varunsh-coder In this case `is-default` will not get fetched as it should be defined inside `action-input`, see below
https://github.com/step-security/secure-workflows/blob/63f710f0dd1f557371a97fe00c6dcf5722d91732/actionmetadata.go#L72-L76
And test case for either `action-input` or `env variable` is already added, see below
https://github.com/step-security/secure-workflows/blob/63f710f0dd1f557371a97fe00c6dcf5722d91732/actionmetadata_test.go#L119-L129